content/html/concepts/favicon/favicon.md

```diff
@@ -0,0 +1,33 @@
+---
+Title: 'Favicon'
+Description: "A favicon is an icon that is displayed in the browser's address bar and tab title for a website."
+Subjects:
+  - 'Web Development'
+  - 'Web Design'
+Tags:
+  - 'Favicon'
+  - 'Documentation'
+CatalogContent:
+  - 'learn-html'
+  - 'paths/front-end-engineer-career-path'
+---
+
+A **favicon** (short for "favorite icon") is an icon that is displayed in the browser's address bar and tab title for a website. It is used to help users quickly identify the website and distinguish it from other tabs or bookmarks.
+
+## Example
+
+To add a favicon to a website, create a small image that will be used as the favicon, and then include a link to it in the `<head>` of the HTML document using the `link` element. Here's an example of how to do this:
+
+```html
+<link rel="shortcut icon" type="image/x-icon" href="/path/to/favicon.ico" />
+```
+
+-The `rel` attribute set to `"shortcut icon"` tells the browser that the linked resource is a favicon.
+
+-The `type` attribute specifies the type of the favicon (in this case, "image/x-icon").
+
+-The `href` attribute specifies the location of the favicon file.
+
+Any image format that is supported by web browsers can be used as a favicon, but the most common format is `.ico`. An `.ico` file can be created using an image editor or online converter tool.
+
+With the favicon file and the `link` element in the HTML, the favicon should be displayed in the browser's address bar and tab title for the website.
```

Comment:
> With the favicon file and the `link` element in the HTML, the favicon should be displayed in the browser's address bar and tab title for the website.

This really depends on the browser being used. It seems that most modern browsers only display the favicon next to the tab title for the website whereas an HTTPS "lock icon" is next to the address bar.

Could we possibly rewrite this line as a new "## Example" section that describes this distinction with the following:

- An \```html code snippet with an image of the Codecademy logo as a favicon.
- A screenshot of what the rendered output would look like on the browser (including the HTTPS lock icon)
  - Be sure to store this screenshot image in the `media/` folder and use the following format for the markdown image: `https://raw.githubusercontent.com/Codecademy/docs/main/media/filename.ext`